ui/src/project.ts

```diff
@@ -13,6 +13,19 @@ export interface Metadata {
   description?: string;
 }
 
+export interface New {
+  type: "new";
```

Comment:
As mentioned this should be an enum. For reference:

https://github.com/radicle-dev/radicle-upstream/blob/1485bba8d343f00e4cb87e3a644db22948ce72d2/ui/src/settings.ts#L2-L5